src/patterns/atoms/forms/errors.hbs

```diff
@@ -28,8 +24,8 @@ labels:
   </div>
 
   <div class="mzp-is-error">
-    <label for="language">Language</label>
-    <select name="language" id="language" required aria-required="true">
+    <label for="language-e">Language</label>
```

Comment:
I can't comment on the line because it didn't change, but the email field above should probably get a new ID too.

Reply:
✅